test/unit_tests/braket/experimental/autoqasm/test_gate_decorator.py

```diff
@@ -110,7 +110,7 @@ def main():
     rx(angle) q;
 }
 qubit[1] __qubits__;
-my_gate(pi / 4) __qubits__[0];"""
+my_gate(0.7853981633974483) __qubits__[0];"""
```

Comment:
This change is also going to affect (+ make uglier, but oh well) various outputs in example notebooks 4, 5, and 6. Can you re-run those notebooks so that the output will be up-to-date with these changes?

Reply:
For it do run on Braket service, it needs to be ugly ...

I can rerun the notebooks

Reply:
Yes, I think it's a fine compromise that it is ugly for now - better that it runs! We can revisit this if/when we rearchitect the way that the oqpy program is generated.